contributed/ggStrongLensingDC2_SL_NanLi_HackurDC2.ipynb

```diff
@@ -0,0 +1,752 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Add galaxy-galaxy strong lenses to DC2\n",
+    "\n",
+    "As is well know, we have Sprinkler to add strong lensed AGNs in DDF. This notebook is for adding galaxy-galaxy strong lensing systems to DC2 in WFD. It is just a starting point, I will keep improving it. Should you have any questions or suggestions please feel free to slack me @linan7788626"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import numpy as np\n",
+    "import pylab as pl\n",
+    "import astropy.io.fits as pyfits\n",
+    "from astropy.table import Table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.cosmology import FlatLambdaCDM\n",
+    "cosmo = FlatLambdaCDM(H0=71, Om0=0.264)\n",
+    "vc = 2.998e5 #km/s\n",
+    "G = 4.3011790220362e-09 # Mpc/h (Msun/h)^-1 (km/s)^2\n",
+    "apr = 206269.43\n",
+    "#--------------------------------------------------------------------\n",
+    "# Cosmology model\n",
+    "#\n",
```

Comment:
It would be great to link this more closely with the CCL, since many of these functions are in that library.

Reply:
Hi Renee, I am in China for a job interview now, and will work on this as soon as I am back in the UK (Feb 21). Sorry for any inconvenience. Cheers, Nan

Reply:
Hi Renee, 

Sorry for the delay, I just replaced astropy with pyccl. 

Cheers
Nan